python/felis/cli.py

```diff
@@ -299,6 +301,22 @@ def merge(files: Iterable[io.TextIOBase]) -> None:
     _dump(normalized)
 
 
+@cli.command("validate")
+@click.argument("files", nargs=-1, type=click.File())
+def validate(files: Iterable[io.TextIOBase]) -> int:
+    """Validate one or more felis YAML files."""
+    rc = 0
+    for file in files:
+        file_name = getattr(file, "name", None)
+        logger.info(f"Validating {file_name}")
+        try:
+            Schema.model_validate(yaml.load(file, Loader=yaml.SafeLoader))
+        except ValidationError as e:
+            logger.error(e)
+            rc = 1
+    return sys.exit(rc)
```

Comment:
```suggestion
    if rc:
        raise click.exception.Exit(rc)
```
I think you are supposed to raise `Exit` here so as not to bypass any other click infrastructure that may be wanting to do some cleanup. (also `return` does nothing because `sys.exit()` immediately exits and also click commands aren't really typed to return anything.

There are a couple of other places that use `sys.exit` that I think need to be fixed.